docs/fundamentals/platform.rst

```diff
@@ -1,2 +1,279 @@
+        results.append(result.array)
+
+And then you can use ``np.hstack`` to stack the obtained results horizontally. By doing this, you would obtain an
+array with shape `(2, N)`, where N is the number of elements inside the loop:
+
+>>> results = np.hstack(results)
+>>> results
+array([[5, 4, 3, 2, 1, 2, 3],
+        [5, 4, 3, 2, 1, 2, 3]])
+TODO: !!! Change this results for the actual ones !!!
+
+You can see how the integrated I/Q values oscillated, indicating that qubit 0 oscillates between ground and
+excited state!
+
+|
+
+A faster Rabi sequence, translating the circuit to pulses:
+-----------------------------------------------------------
+
+Since you are looping over variables that are independent of the circuit (in this case, the gain of the AWG),
+you can speed up the experiment by translating the circuit into pulses only once:
+
+.. code-block:: python3
+
+    from qililab.pulse.circuit_to_pulses import CircuitToPulses
+
+    pulse_schedule = CircuitToPulses(platform=platform).translate(circuits=[circuit])
+
+and then, executing the obtained pulses inside the loop, by passing the translated
+``pulse_schedule`` instead than the ``circuit``, to the ``execute()`` method:
+
+.. code-block:: python3
+
+    results = []
+    gain_values = [0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.9, 1.0]
+
+    for gain in gain_values:
+        platform.set_parameter(alias="drive_q", parameter=ql.Parameter.GAIN, value=gain)
+        result = platform.execute(program=pulse_schedule, num_avg=1000, repetition_duration=6000)
+        results.append(result.array)
+
+This approach yields to similar results, but much faster!
+
+>>> results = np.hstack(results)
+>>> results
+array([[5, 4, 3, 2, 1, 2, 3],
+        [5, 4, 3, 2, 1, 2, 3]])
+TODO: !!! Change this results for the actual ones !!!
+
+|
+
+Ramsey sequence, looping over a parameter inside the circuit:
+----------------------------------------------------------------
+
+To perform a Ramsey sequence, build, connect and setup the platform as before, but this time with a different circuit:
+
+.. code-block:: python
+
+    import qililab as ql
+
+    from qibo.models import Circuit
+    from qibo import gates
+
+    # Defining the Ramsey circuit:
+    circuit = Circuit(q + 1)
+    circuit.add(gates.RX(q, theta=np.pi/2))
+    circuit.add(ql.Wait(q, t=0))
+    circuit.add(gates.RX(q, theta=np.pi/2))
+    circuit.add(gates.M(q))
+
+    # Building the platform:
+    platform = ql.build_platform(runcard="runcards/galadriel.yml")
+
+    # Connecting and setting up the platform:
+    platform.connect()
+    platform.initial_setup()
+    platform.turn_on_instruments()
+
+where you would add two default qibo ``RX`` gates, with a qililab ``Wait`` gate in between, which is just a personalized qibo gate that adds a
+free evolution of duration ``t`` that corresponds to a rotation at the detuning frequency, around the Z axis:
+
+.. image:: platform_images/ramsey_bloch.png
```

Comment:
Love the drawings *_*

Reply:
Thankyou! I think they can be improved, but for now are pretty decent hehe! 🙌